sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -703,6 +703,18 @@ def get_module_media_interface(self):
         else:
             return 'Unknown media interface'
 
+    def is_copper(self):
```

Comment:
@jcsteven why not use `is_flat_memory()` https://github.com/sonic-net/sonic-platform-common/blob/174bbd47f0c5a3b8bd181f79b9e700ffa38906f6/sonic_platform_base/sonic_xcvr/api/public/cmis.py#L370